src/Utils/request/errorHandler.ts

```diff
@@ -32,11 +32,19 @@ export function handleHttpError(error: Error) {
 
   if (isBadRequest(error)) {
     const errs = cause?.errors;
-    if (isPydanticError(errs)) {
+    if (errs && isPydanticError(errs)) {
       handlePydanticErrors(errs);
       return;
     }
-    Notifications.BadRequest({ errs });
+    if (cause && typeof cause === "object" && !Array.isArray(cause)) {
+      const firstKey = Object.keys(cause)[0];
+      const firstError = firstKey ? cause[firstKey] : null;
+      if (firstError && Array.isArray(firstError)) {
+        toast.error(firstError[0]);
+        return;
+      }
```

Comment:
why limit to just first error? what if backend gives more errors?

also move this to a separate function like other checks and handlers. let's not make the global error handler messy by inlining things. not readable code.